src/lib.rs

```diff
@@ -2,6 +2,66 @@
 // See the COPYRIGHT file at the top-level directory of this distribution.
 // Licensed under the MIT license, see the LICENSE file or <http://opensource.org/licenses/MIT>
 
+//! # Cairo bindings
+//!
+//!   This library contains safe Rust bindings for [Cairo](https://www.cairographics.org/).
```

Comment:
Why the two extra whitespaces at the beginning?

Reply:
Readability?
But if the convention differs, I will remove them.

Reply:
It's markdown so they'll get removed. And it makes the doc comment looking strange. It'd be nice to remove then if you don't mind. :)

Reply:
Done!
I removed quite a lot of whitespace.
If you want some back just tell me.